assets/metrics-server/api-service.yaml

```diff
@@ -16,5 +16,6 @@ spec:
   service:
     name: metrics-server
     namespace: openshift-monitoring
+    port: 443
```

Comment:
maybe in the future we can change `ApplyAPIService` so it takes the default value of the port (443) into consideration
(a warning would be useful as well for other users)